include/seqan/basic/debug_test_system.h

```diff
@@ -829,16 +833,17 @@ int endTestSuite()
 {
     delete[] StaticData::basePath();
 
+    // Delete all temporary files that still exist.
+    for (unsigned i = 0; i < StaticData::tempFileNames().size(); ++i)
+        if (_deleteTempFile(StaticData::tempFileNames()[i]))
+            ++StaticData::errorCount();
```

Comment:
It seems that if the file could be deleted the error count will be increased, but it should increase the errorCount if it is not able to delete.
Is there a ```!``` missing before ```_deleteTmpFile```?